test/integration/command-logging-and-monitoring/command_logging.spec.test.ts

```diff
@@ -0,0 +1,8 @@
+import * as path from 'path';
+
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+describe('Command Logging Spec (unified)', () => {
+  runUnifiedSuite(loadSpecTests(path.join('command-logging-and-monitoring', 'logging', 'command')));
```

Comment:
Going to change this to actually run all of the command logging tests, but for only trying to run one file